tests/e2e-tests/test-devicestate.sh

```diff
@@ -48,19 +52,25 @@ test_result=1 # fail by default
 
 # The Central app is stored in PipelineResources-IoTEdgeConfig resource group
 echo Get access token to subscription "Azure IoT DDE team subscription"
-armToken=$(az account get-access-token --resource https://apps.azureiotcentral.com -s ${subscription} | jq -r '.accessToken')
+out=$(az account get-access-token --resource https://apps.azureiotcentral.com -s ${subscription})
+echo $out
+armToken=$(jq -r '.accessToken' <<< "$out")
 
 # Create API token
 echo Create API token to interact with Central app "https://e2etest-iotc-iiot-asset-app.azureiotcentral.com"
-apiToken=$(curl -X PUT -d '{"roles":[{"role":"ca310b8d-2f4a-44e0-a36e-957c202cd8d4"}]}' -H "Content-Type:application/json" -H "Authorization:Bearer $armToken" https://e2etest-iotc-iiot-asset-app.azureiotcentral.com/api/preview/apiTokens/${token_id} 2>/dev/null | jq -r '.token');
+out=$(curl -X PUT -d '{"roles":[{"role":"ca310b8d-2f4a-44e0-a36e-957c202cd8d4"}]}' -H "Content-Type:application/json" -H "Authorization:Bearer $armToken" https://e2etest-iotc-iiot-asset-app.azureiotcentral.com/api/preview/apiTokens/${token_id});
+echo $out
+apiToken=$(jq -r '.token' <<< "$out")
 
 if [ "$apiToken" == "null" ]; then 
```

Comment:
nit - inconsistent placement of 'then' compared to other code above (see line 29 for example).
     - other 'infractions' below...